src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -123,7 +276,53 @@ impl PrivilegeAccess {
             }
         }
 
-        session.validate_privilege(object, privileges).await
+        match session.validate_privilege(object, privileges.clone()).await {
+            Ok(_) => Ok(()),
+            Err(_) => {
```

Comment:
IMHO we'd better to have a check about the error type here, besides privilege denied error, there might also other kind of errors like metasrv connection error, etc.

Reply:
Get

Reply:
Done